tasks/classes/classes.js

```diff
@@ -6,22 +6,38 @@
  */
 //put your code here
 
+class Person {
+    constructor(firstName, lastName) {
+        this.firstName = firstName,
+            this.lastName = lastName;
+    }
+    getFullName() {
+        return this.firstName + " " + this.lastName;
+    }
+
```

Comment:
you did not close properly the class with } bracket.

Reply:
Thanks a lot Dima! Notes taken

![image](https://user-images.githubusercontent.com/62101674/79331602-1bca1480-7f24-11ea-97a4-a6f6b3a5250f.png)

Reply:
Please approve